documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -568,6 +568,13 @@ During the "partition walk" `autoboot.txt` files are not processed to avoid cycl
 
 Default: `0`
 
+[[BOOTVAR0]]
+==== `BOOTVAR0`
+
+Allows setting the conditional variable bootvar0 used by `config.txt`. See xref:../config_txt/conditional.adoc#bootvar0[bootvar0].
```

Comment:
Need to make this change to make the build (CI) work:
```suggestion
Allows setting the conditional variable bootvar0 used by `config.txt`. See xref:config_txt.adoc#bootvar0[bootvar0].
```
(this is because of the way that the nested includes work)

Reply:
Yeah, unfortunately there's a difference in links required to view the raw asciidoc on GitHub, and the links required for the [rendered HTML output](https://www.raspberrypi.com/documentation/).
https://github.com/raspberrypi/documentation/blob/develop/CONTRIBUTING.md explains how you can preview the HTML output locally.